### DISCUSSION TITLE
I hate this!!!

### DISCUSSION THREAD

**Discussion by headrotor** [community/community#54046] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

No matter how hard you try, you are not a social network. Adding useless crufty features is a sign of desperation. 

Tell your managers who requested this that it is a mistake. 

Anyway thanks for reminding me there are wonderful open-source alternatives, I have just created an account at [https://codeberg.org/](https://codeberg.org) -- everyone reading this should create one too because github with it's useless feature creep, is doomed.

**Comment by csilverman** [comment]
Did not know about Codeberg. Thank you. I never had any real reason to look for GitHub alternatives before, but this latest abomination is making me seriously rethink that.
